pkg/model/components/etcdmanager/model.go

```diff
@@ -301,6 +303,13 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster *kops.EtcdClusterSpec) (*v1.Po
 	if backupStore == "" {
 		return nil, fmt.Errorf("backupStore must be set for use with etcd-manager")
 	}
+	// We need to ensure that the backupStore passed to etcd-manager contains
+	// the suffix "backups/etcd/{etcd-cluster-name}" as this is hardcoded
+	// in pkg/model/components/etcdmanager/model.go when generating etcd-cluster-spec
```

Comment:
So I think we _could_ require that if the user specified the backupStore location, we just add `control/etcd-cluster-spec` to it, and we assume that they have configured main & events separately.  But it's so easy to mess up and configure them with the same value, that I think it's better to do what you've done here.

One thing that might work ... we could validate that the paths are in fact different from each other?